{
  "config": {
    "step": {
      "user": {
        "data": {
          "description": "Log in with your Jätekukko Omakukko credentials.",
          "customer_number": "Customer number",
          "password": "[%key:common::config_flow::data::password%]"
        }
      },
      "reauth_confirm": {
        "data": {
          "description": "Re-authenticate with your Jätekukko Omakukko credentials.",
          "customer_number": "Customer number",
          "password": "[%key:common::config_flow::data::password%]"
        }
      }
    },
    "error": {
      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
      "unknown": "[%key:common::config_flow::error::unknown%]"
    },
    "abort": {
      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
      "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]",
      "reauth_failed_existing": "Could not update the config entry, please remove the integration and set it up again."
    }
  }
}
